docs/DeveloperGuide.md

```diff
@@ -61,6 +61,81 @@ When the user launches the app, the main program will initialize a `TopicsStorag
 
 ![TopicsStorage load](./images/TopicsStorage_load.png)
 
+###Implementation of commands relating to TopicList, namely:
```

Comment:
This header isn't showing up properly I think you need to add a space after the # , noticed this in a few other places as well